src/analytics/private/load-ods-data-workflow.ts

```diff
@@ -177,15 +180,138 @@ export class LoadOdsDataToRedshiftWorkflow extends Construct {
     });
 
     //
-    // function:  createLoadRedshiftTableWorkflow
+    // Refresh materialized views
     //
-    const createLoadRedshiftTableWorkflow = (odsTableName: string) => {
+    const refreshViewsFn = this.refreshViewsFn(props, copyRole);
+    const refreshViewsStep = new LambdaInvoke(this, `${this.node.id} - Refresh materialized views`, {
+      lambdaFunction: refreshViewsFn,
+      resultSelector: {
+        'Payload.$': '$.Payload',
+      },
+      resultPath: '$.refreshViewsOut',
+    });
+    refreshViewsStep.addRetry({
+      errors: ['Lambda.TooManyRequestsException'],
+      interval: Duration.seconds(3),
+      backoffRate: 2,
+      maxAttempts: 6,
+    });
+
+    //
+    // Next state machines
+    //
+    let nexTaskExecChain;
+    for (const nexTask of props.nextStateStateMachines) {
+      const taskName = nexTask.name;
+      const stateMachine = nexTask.stateMachine;
+      const input = nexTask.input;
+      const integrationPattern = nexTask.integrationPattern?? IntegrationPattern.REQUEST_RESPONSE;
+      const resultPath = `$.${nexTask.resultPath ?? taskName.replace(/ /g, '')}`;
+      const nexTaskExec = new StepFunctionsStartExecution(this, `${this.node.id} - ${taskName}`, {
+        stateMachine,
+        integrationPattern,
+        input,
+        resultSelector: {
+          'ExecutionArn.$': '$.ExecutionArn',
+        },
+        resultPath,
+      });
+      if (!nexTaskExecChain) {
+        nexTaskExecChain = nexTaskExec;
+      } else {
+        nexTaskExecChain = nexTaskExecChain.next(nexTaskExec);
+      }
+    }
+
+    const allCompleted = new Succeed(this, `${this.node.id} - All Completed`);
+    if (nexTaskExecChain) {
+      nexTaskExecChain.next(allCompleted);
+    } else {
+      nexTaskExecChain = allCompleted;
+    }
+
+    //
+    // Main workflow
+    //
+    const logGroup = createLogGroup(this,
+      {
+        prefix: `/aws/vendedlogs/states/Clickstream/LoadData-${this.node.id}`,
+      },
+    );
+
+    let parallelLoadDataToTables = new Parallel(this, `${this.node.id} - Load data to tables`);
+    const subWorkflow = this.createSubWorkflow(ddbTable, props, copyRole, logGroup);
+    for (const odsTable of Object.keys(props.tablesOdsSource)) {
+      const subExecution = new StepFunctionsStartExecution(this, `${this.node.id} - ${odsTable}`, {
```

Comment:
specify the name, like the below pattern for easy debugging,

`<parent execution id>-<table name>`

Reply:
changed to <parent execution name>-<table name>.
The execution id is too long.